### ISSUE TITLE
No way to close documents

### ISSUE THREAD

**Issue by angelsix** [careteditor/caret#460]
If I open up a document, there is no way to close it. 

If I start a new document, I can tab between documents, but again, no way to close any of them. 

I want to re-open Caret and have it at a new document if nothing was opened before, however due to the fact you can never close a document, it always opens with whatever you last worked on, which I do not want.

**Comment by netmasteroc3** [comment]
I too would love to have caret operate more like a standard text editor/application.

Start it from the application (or application shortcut) you get a blank document. Start it from an associated file, it opens that file.

**Comment by astoilkov** [comment]
Currently it is possible to open a new untitled file with `File -> New` which is the default/initial state when you first open the app. We can make this more familiar by mapping the `Cmd/Ctrl+W` shortcut (standard shortcut for closing a document in many editors) to do the same thing as `File -> New`.

What do you think?

**Comment by angelsix** [comment]
That's better than it is now but it still really needs a proper file open close function IMO

**Comment by astoilkov** [comment]
How do you think the file close operation should work?

Remove the file from the `Ctrl+Tab` window and open the previously opened file and when there are no files left in the `Ctrl+Tab` window show a blank window with no file opened (not even untitled)?

**Comment by angelsix** [comment]
You would do it like Word, VS Code, Browsers and all other apps do. With nothing open you would see the welcome page perhaps a few buttons saying to open a document, create new one etc...

**Comment by nickbe** [comment]
I agree completely with that. This feeld quite strange and I'm still not used to it. If I close a document then at least 2 simple button in the windows center would be nice: "Create new document", "Reopen last document"

**Comment by BsNoSi** [comment]
I´d prefer the usual behaviour of any coding editor. If the file is not saved – ask for saving or not. Then  close it and behave like described three entries above from astoilkov.
